### PR TITLE
New sign-in flow and content fallback gherkins

### LINKED ISSUE #6904: Automated tests and gherkin stories still needed for #6865
We didn't have time to do automated tests or gherkin stories for https://github.com/learningequality/kolibri/pull/6865 -- but if would be good to have these before this ships in 0.14, even if it's still a hidden feature at that point.

### PR DESCRIPTION
**Pull request by radinamatic**
### Summary
- Modifies the sign in testing scenarios for the new user flow
- Fixes #6904 

### Reviewer guidance
…

### References
…

----

### Contributor Checklist

PR process:

- [ ] PR has the correct target branch and milestone
- [ ] PR has 'needs review' or 'work-in-progress' label
- [ ] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

Testing:

- [ ] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

### Reviewer Checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

**Review by indirectlylit — APPROVED** [review]
story looks correct to me

**Comment by indirectlylit** [comment]
still WIP or shall we merge as-is?

File: integration_testing/features/app-context/app-context-manage-sign-in-options.feature [hunk 1]
```diff
@@ -50,11 +56,96 @@ Feature: Manage sign in options within app context
...
+		Then I see *Sign into '<facility>'*
+			And I see a list of usernames
+		When I tap my <username>
+		Then I see *Sign into '<facility>' as '<username>'*
+			And I see the input field for entering my password
+			And I type my password
+			And I click on *Sign in*
+		Then I see my *Learn* page
+
+	Scenario: Facility name will not be displayed on the sign in page if there is only one facility on the device and the use context is personal
```
Comment (radinamatic) [inline integration_testing/features/app-context/app-context-manage-sign-in-options.feature:123]: @nucleogenesis This scenario is verbatim from Riddhi's gherkins on Notion, but I still see the facility name when I install on Android device with **Quick setup** (which should be how the use is recognized as _personal_, correct?)

DevTools - localhost:8080-en-user-_882

Comment (radinamatic) [reply integration_testing/features/app-context/app-context-manage-sign-in-options.feature:123]: Ok, issue already reported here: https://github.com/learningequality/kolibri/issues/7303#issuecomment-661272301

**Comment by radinamatic** [comment]
Closed in favor of #7358.

**Pull request closed without merging**